src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/04-Travis-CI-with-sbt.md

```diff
@@ -175,6 +175,7 @@ cache:
   directories:
     - \$HOME/.ivy2/cache
     - \$HOME/.sbt
+    - \$HOME/.coursier
```

Comment:
Unfortunately https://github.com/scala/scala-xml/pull/357 is incorrect. 
See https://github.com/sbt/website/pull/818 for the update.